net/sched/sch_skbprio.c

```diff
@@ -176,7 +176,7 @@ static int skbprio_change(struct Qdisc *sch, struct nlattr *opt,
 	const unsigned int min_limit = 1;
 
 	if (ctl->limit == (typeof(ctl->limit))-1)
```

Comment:
Replace this whole `if` statement into `sch->limit = ctl->limit;`. This is possible because we'll work with any value `sch->limit` gets